app/components/services/disclosure/index.spec.js

```diff
@@ -13,31 +13,76 @@ describe('disclosureApi', function() {
     disclosureApi = _disclosureApi_;
   }));
 
-  it('exists', function() {
-    expect(disclosureApi).to.be.ok;
-  });
-
-  it('has contributions', function() {
-    expect(disclosureApi).to.have.property('contributions');
-  });
 
-  it('has elections', function() {
-    expect(disclosureApi).to.have.property('elections');
+  // locality.search
+  it('get a locality', function() {
+    // Search for any locality
+    disclosureApi.locality.search({q: ''})
+      .then(function(localities) {
+        localities.each(function(locality) {
+          expect(locality).to.have.property('name');
+          expect(locality).to.have.property('type');
+          expect(locality).to.have.property('id');
+        });
+      });
   });
 
-  it('has locations', function() {
-    expect(disclosureApi).to.have.property('locations');
+  // locality.current_ballot
+  it('get a ballot', function() {
+    // Search for any locality
+    disclosureApi.locality.search({q: ''})
+      .then(function(localities) {
+        // Get the current ballot for the first locality
+        var locality = localities[0];
+        disclosureApi.locality.current_ballot({locality_id: locality.id})
+          .then(function(ballot) {
+            expect(ballot).to.have.property('locality_id');
+            expect(ballot).to.have.property('ballot_id');
+            expect(ballot).to.have.property('ballot_items');
+          });
+      });
```

Comment:
I would just use a known-good locality id. Otherwise it's testing too much IMHO.